docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md

```diff
@@ -75,6 +75,14 @@ install -v -m 755 ./zm_rpcapi-bsd /usr/local/etc/rc.d/zm_rpcapi
 ```
 
 
+## Cleaning old files
+
+Run
+```sh
+cd $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')
+sudo rm patch_*.pl
+```
```

Comment:
* `$(...)` works in Bash, Bourne shell and some other shells, but not in csh, the default FreeBSD root shell.
* sudo() does not work in FreeBSD.

````
Run in Linux:
```sh
sudo rm $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')/patch_*.pl
```
Run in FreeBSD:
```sh
rm `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')`/patch_*.pl
```
````

Reply:
fixed